### PR TITLE
Hyundai: fix wheelbase and steeringratio for Kona, Kona HEV , Kona EV

### PR DESCRIPTION
**Pull request by haram-KONA**
Update interface.py to fix " wheelbase and steering ratio " for Kona HEV (Kona, Kona EV)

( https://www.hyundai.news/eu/articles/press-releases/all-new-hyundai-kona-hybrid-technical-specifications.html )

Dongl ID : 0665dbbc79ef999e
Route: 0665dbbc79ef999e|2022-05-04--14-47-57
(test drive  by Kona Hybrid 2020)

### CODE REVIEW

**Comment by pd0wm** [comment]
Does the Kona have variable ratio steering? The learned steer ratio in the posted drive is actually pretty close to 13.4

**Comment by haram-KONA** [comment]
yes thats right




내 Galaxy에서 보냄



-------- 원본 이메일 --------
발신: Willem Melching ***@***.***>
날짜: 22/5/12 20:56 (GMT+09:00)
받은 사람: commaai/openpilot ***@***.***>
참조: haram-KONA ***@***.***>, Author ***@***.***>
제목: Re: [commaai/openpilot] Hyundai: fix wheelbase and steeringratio for Kona, Kona HEV , Kona EV (PR #24464)


Does the Kona have variable ratio steering? The learned steer ratio in the posted drive is actually pretty close to 13.4

**Comment by haram-KONA** [comment]
KONA Steer Ratio Values is not variable but fixed

Windows용 메일<https://go.microsoft.com/fwlink/?LinkId=550986>에서 발송된 메일입니다.

________________________________
보낸 사람: Willem Melching ***@***.***>
보낸 날짜: Thursday, May 12, 2022 8:56:46 PM
받는 사람: commaai/openpilot ***@***.***>
참조: haram-KONA ***@***.***>; Author ***@***.***>
제목: Re: [commaai/openpilot] Hyundai: fix wheelbase and steeringratio for Kona, Kona HEV , Kona EV (PR #24464)


Does the Kona have variable ratio steering? The learned steer ratio in the posted drive is actually pretty close to 13.4

File: selfdrive/car/hyundai/interface.py [hunk 1]
```diff
@@ -123,10 +123,10 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[], disabl
       ret.lateralTuning.indi.actuatorEffectivenessV = [2.3]
       ret.minSteerSpeed = 60 * CV.KPH_TO_MS
     elif candidate in (CAR.KONA, CAR.KONA_EV, CAR.KONA_HEV):
-      ret.lateralTuning.pid.kf = 0.00005
+      ret.lateralTuning.pid.kf = 0.000055
```
Comment (pd0wm) [inline selfdrive/car/hyundai/interface.py:126]:
```suggestion
      ret.lateralTuning.pid.kf = 0.00005
```

Comment (pd0wm) [reply selfdrive/car/hyundai/interface.py:126]: This has to go in a different PR.

**Pull request merged**